### PR TITLE
Enhancements

### PR DESCRIPTION
**Pull request by praveen4g0**
* New assertion to operator installation.
(clusterRole, configmaps injection assertions)
* Add test to triggers to verify tiggerbinding parsing event body completely/via marshalJsonCEL function.

### CODE REVIEW

File: specs/triggers/triggerbinding.spec [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+Pre condition:
+  * Validate Operator should be installed
+
+## Verify CEL marshaljson function Test: PIPELINES-06-TC01
+Tags: e2e,triggers,non-admin
+Component: Triggers
+Level: Integration
+Type: Functional
+Importance: Critical
```
Comment (VeereshAradhya) [inline specs/triggers/triggerbinding.spec:11]: Are we planning to run this as part of acceptance tests?

File: specs/triggers/triggerbinding.spec [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# Verify triggerbindings spec
+
+Pre condition:
+  * Validate Operator should be installed
+
+## Verify CEL marshaljson function Test: PIPELINES-06-TC01
```
Comment (VeereshAradhya) [inline specs/triggers/triggerbinding.spec:6]: The test cases with ID `PIPELINES-06` series already present, use `PIPELINES-10-TC01` and add `PIPELINES-10` in the first line of the file

File: specs/triggers/tutorial.spec [hunk 3]
```diff
@@ -4,7 +4,7 @@ PIPELINES-06
 Pre condition:
   * Validate Operator should be installed
 
-## Run Triggers tutorial (by Automatically configuring users webhook to git repo): PIPELINES-06-TC01
+## Run Triggers tutorial (by Automatically configuring users webhook to git repo): PIPELINES-07-TC01
```
Comment (VeereshAradhya) [inline specs/triggers/tutorial.spec:7]: Don't change the existing test cases. Let's keep this test case as `PIPELINES-06-TC01`

File: specs/triggers/v1alpha1.spec [hunk 4]
```diff
@@ -4,7 +4,7 @@ PIPELINES-07
 Pre condition:
   * Validate Operator should be installed
 
-## v1alpha1 resources creation Test: PIPELINES-07-TC01
+## v1alpha1 resources creation Test: PIPELINES-08-TC01
```
Comment (VeereshAradhya) [inline specs/triggers/v1alpha1.spec:7]: same here.. don't change the test case IDs

File: specs/triggers/triggerbinding.spec (hunk 1 above)
Comment (praveen4g0) [reply specs/triggers/triggerbinding.spec:11]: yes

File: specs/triggers/triggerbinding.spec (hunk 2 above)
Comment (praveen4g0) [reply specs/triggers/triggerbinding.spec:6]: in that case shall I change it 09 ?? instead of 10

File: pkg/operator/operator.go [hunk 5]
```diff
@@ -18,7 +18,12 @@ func ValidateRBAC(cs *clients.Clients, rnames config.ResourceNames) {
 	EnsureTektonConfigStatusInstalled(cs.TektonConfig(), rnames)
 	//Verify `pipelineSa` is created in any namespace
 	AssertServiceAccount(cs, store.Namespace(), "pipeline")
-	//Verify roleBindings are created in any namespace
+	//Verify clusterrole
+	AssertClusterRole(cs, "pipelines-scc-clusterrole")
+	//Verify configmaps is created in any namespace
+	AssertConfigMap(cs, store.Namespace(), "config-service-cabundle")
```
Comment (VeereshAradhya) [inline pkg/operator/operator.go:24]: In the future, if we want to verify a different configmap that is not part of another test case, the step `Validate RBAC` cannot be used right? Isn't it better to have a different step for verifying the configmaps? WDYT?

Comment (praveen4g0) [reply pkg/operator/operator.go:24]: makes sense I would rather rename it to `validateOperatorRBAC` WDYS??

Comment (VeereshAradhya) [reply pkg/operator/operator.go:24]: +1, please change the step as well in test case

Comment (praveen4g0) [reply pkg/operator/operator.go:24]: I guess it makes sense to keep name as expected as it is under package operator.

**Review by VeereshAradhya — APPROVED** [review]

**Pull request merged**